src/ejabberd_config.erl

```diff
@@ -1127,9 +1127,39 @@ compute_config_version(LC, LCH) ->
     crypto:hash(sha, term_to_binary(L1)).
 
 compute_config_file_version(#state{opts = Opts, hosts = Hosts}) ->
-    L = sort_config(Opts ++ Hosts),
+    Opts2 = filter_out_node_specific_options(Opts),
+    L = sort_config(Opts2 ++ Hosts),
     crypto:hash(sha, term_to_binary(L)).
 
+filter_out_node_specific_options([]) ->
+    [];
+filter_out_node_specific_options([{local_config, {modules, Host}, Mods} | Opts]) ->
+    NewMods = lists:foldl(fun(Path, Mods) -> delete_path_in_proplist(Mods, Path) end,
+                          Mods, node_specific_module_options()),
+    [{local_config, {modules, Host}, NewMods} | Opts];
+filter_out_node_specific_options([Opt | Opts]) ->
+    [Opt | filter_out_node_specific_options(Opts)].
+
+% @doc The list of options that should not be checked if equal
+% between nodes in cluster. Each option is expressed as
+% a list of keys in proplist that should be deleted (if such
+% path exists). The `root path` for them is a list of modules.
+node_specific_module_options() ->
```

Comment:
Hmm, can this path go into ejabberd.cfg itself?
I don't like it to be hardcoded.
It feels extremely installation-specific. For different clients we can have different per-node parameters.

I even think there would be installations with the same endpoints for all nodes.

Reply:
I'm not sure if it's not too meta. These hardcoded are the options that MIGHT be different and we don't require them to be the same (but they might be). It is not user's role to define them, I think.

Reply:
IMO can be done as a separate task. This PR solves immediate issue.

Reply:
@kanes115 I agree with @arcusfelis. It wouldn't be too meta and we already have config options only for advanced users, so why not affect the `reload_cluster` logic?

Reply:
Yea, I agree